js/BonusWheel.js

```diff
@@ -1,11 +1,18 @@
-// class BonusWheel extends Wheel {
-//   constructor() {
-//     super();
-//     this.bonusPlayer = null;
-//     this.bonusWheelValue = null;
-//   }
-// }
+class BonusWheel extends Wheel {
+  constructor() {
```

Comment:
I think we need to pass in the properties from the wheel in the constructor as well as in the super to technically "inherit" the class.